### PR TITLE
Keep AI Assistant button visible when panel open, in active state

### PR DESCRIPTION
**Pull request by lukemelia**
https://github.com/cardstack/boxel/assets/353/26687068-3d74-4e2e-9b6e-4e29d2014afd

### CODE REVIEW

File: packages/host/app/components/operator-mode/submode-layout.gts [hunk 1]
```diff
@@ -165,33 +167,35 @@ export default class SubmodeLayout extends Component<Signature> {
...
             @submode={{this.operatorModeStateService.state.submode}}
             @onSubmodeSelect={{this.updateSubmode}}
             class='submode-switcher'
           />
           {{yield this.openSearchSheetToPrompt}}
+          <AiAssistantButton
+            class='chat-btn'
+            @isActive={{this.isAiAssistantVisible}}
+            {{on 'click' this.toggleChat}}
+          />
```
Comment (backspace) [inline packages/host/app/components/operator-mode/submode-layout.gts]: This change means the button shows even when the AI bot isn’t running, it just does nothing, which seems incorrect to me.

screencast 2024-03-20 15-52-01

File: packages/boxel-ui/addon/src/components/resizable-panel-group/usage.gts [hunk 2]
```diff
@@ -158,28 +148,23 @@ export default class ResizablePanelUsage extends Component {
...
             </ResizablePanel>
             <ResizeHandle />
             <ResizablePanel
               @defaultLengthFraction={{this.verticalPanel2DefaultHeightFraction}}
               @minLengthPx={{this.verticalPanel2MinHeightPx}}
-              style={{cssVar
-                boxel-panel-resize-handler-width=this.boxelPanelResizeHandleWidth.value
-                boxel-panel-resize-handler-background-color=this.boxelPanelResizeHandleBackgroundColor.value
-                boxel-panel-resize-handler-hover-background-color=this.boxelPanelResizeHandleHoverBackgroundColor.value
-              }}
```
Comment (backspace) [inline packages/boxel-ui/addon/src/components/resizable-panel-group/usage.gts:182]: Nice consolidation 🎉

Comment (backspace) [inline packages/boxel-ui/addon/src/components/resizable-panel-group/usage.gts:182]: Nice consolidation 🎉

**Review by backspace — CHANGES_REQUESTED** [review]

File: packages/host/app/components/operator-mode/submode-layout.gts (hunk 1 above)
Comment (lukemelia) [reply packages/host/app/components/operator-mode/submode-layout.gts]: Whoops -- good catch

**Review by backspace — APPROVED** [review]

**Review by FadhlanR — APPROVED** [review]
Looks good!

**Pull request merged**